pg_search/src/postgres/delete.rs

```diff
@@ -24,14 +24,15 @@ use crate::index::reader::index::SearchIndexReader;
 use crate::postgres::storage::buffer::BufferManager;
 use crate::postgres::storage::metadata::MetaPage;
 
+use crate::postgres::rel::PgSearchRelation;
 use anyhow::Result;
-use pgrx::{pg_sys, PgRelation};
+use pgrx::pg_sys;
 use tantivy::index::SegmentId;
 use tantivy::indexer::delete_queue::DeleteQueue;
 use tantivy::indexer::{advance_deletes, DeleteOperation, SegmentEntry};
 use tantivy::{Directory, DocId, Index, IndexMeta, Opstamp};
 
-#[pg_guard]
+// #[pg_guard]
```

Comment:
TODO?

Reply:
Ooof.  That was a temporary change to get RustRover out of its analyzer slump!  I'll fix